### ISSUE TITLE
Allow multiple connections from the chassis pipe

### ISSUE THREAD

**Issue by Armigus** [RS485/LogisticsPipes#1100]
Limiting logistics pipes to one connection per pipe is sometimes harmful, especially when you want to have a dense collection of high-capacity, single item containers such as Deep Storage Units and Bedrockium drums.  The proposed functionality could work both as a pipe and as a module.

I can work around it for now but please take this seriously.

**Comment by bookerthegeek** [comment]
That is not the way pipes work.

**Comment by davboecki** [comment]
Chassi Pipes are only ment for one connection. If you want to have multiple connections on one functionality you need to use the pipe version. So in your case the Basic Pipe. I personaly feel that that is a good balance.

**Comment by hron84** [comment]
I have to say not all module has a pipe version (like, advanced extraction modules).

**Comment by davboecki** [comment]
Yes that is correct. But in most cases where there is only a module, the functionality is only intended for one connection. 
It's a question of balance and design decision, and we chose to keep the original design of a chassi pipe that is connected to only one chest.
If you fell the need for a specific module functionality to be available in pipe form (so that it can be used on multiple inventories at once), feel free to ask for that specific module to be added in pipe form as well.

**Comment by Armigus** [comment]
I was looking for an upgrade to allow multiple connections.  That way it can be used for a variety of pipes including liquid and bee variants.
